MANIFEST.in

```diff
@@ -2,4 +2,5 @@ include LICENSE
 include README.rst
 
 recursive-include src/vivarium_nih_us_cvd *.py *.yaml *.ipynb
+recursive-include src/vivarium_nih_us_cvd/data *.csv
```

Comment:
Why are we doing this at all?

Reply:
So that when we `pip install` we get those datasets. (`pip install -e` will include everything)

Reply:
Why ever `pip install` non-editably a research repo?